src/docs/markdown/caddyfile/directives/reverse_proxy.md

```diff
@@ -852,3 +855,16 @@ example.com {
 ```
 
 
+Using [active health checks](#active-health-checks) and `health_upstream` can be helpful when creating an intermediate service to do a more thorough health check. `{http.reverse_proxy.active.target_upstream}` can then be used as a header to provide the original upstream to the health check service.
+
+```caddy
+example.com {
+	reverse_proxy node1:80 node2:80 node3:80 {
+		health_uri /health
+        health_upstream 127.0.0.1:53336
+        health_headers {
+          Full-Upstream {http.reverse_proxy.active.target_upstream}
+        }
```

Comment:
The indentation here isn't aligned.

Reply:
Fixed, thanks!

Reply:
The indentation is still wrong. Use tabs, not spaces.

Reply:
Hmm. I did use tabs, the browser editor must be auto correcting it.